docs/user_guide/hopsworks/newProject.rst

```diff
@@ -65,3 +65,5 @@ SMALLINT, TABLE, TABLESAMPLE, THEN, TIMESTAMP, TO, TRANSFORM, TRIGGER, TRUE, TRU
 UNIQUEJOIN, UPDATE, USER, USING, UTC_TMESTAMP, VALUES, VARCHAR, WHEN, WHERE, WINDOW, WITH, COMMIT, ONLY,
 REGEXP, RLIKE, ROLLBACK, START, CACHE, CONSTRAINT, FOREIGN, PRIMARY, REFERENCES, DAYOFWEEK, EXTRACT, FLOOR,
 INTEGER, PRECISION, VIEWS, TIME, NUMERIC, SYNC.
+
+And any word containing _FEATUESTORE.
```

Comment:
instead of word maybe name?
also can we specify it is case insensitive for all of the above... so that it is clear that any of: 
* _FEATUSTORE 
* _featurestore
* _featureStore 
will not work

Reply:
I think that is too much. Let the user figure it out.